src/App.php

```diff
@@ -94,7 +94,9 @@ public function getContainer()
     private function getDispatcher()
     {
         $container = $this->container;
-
+        if (!is_callable($container->get("dispatcher"))) {
+            throw new \RuntimeException("Dispatcher must be a callable");
+        }
```

Comment:
we need to have consistency for using double quote or single quote, I prefer single quote